spec/facades/photo_facade_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe "Photo Facade" do
```

Comment:
these tests seem more in line with what we would test in a service, not a facade. 

In the Facade I believe we want to test what the facade will give the Controller (or hand to a POROS to then hand a product to a Controller. In other words, what the Facade is _putting out_.